### ISSUE TITLE
Emulator running as root user exhibits bad behavior

### ISSUE THREAD

**Issue by DavidPerezIngeniero** [budtmo/docker-android#223]
## 🐛 Bug Report

Operating System: 
Ubuntu 18.04

Docker Image:
budtmo/docker-android-x86-9.0:1.8-p12

Docker Version:  
19.03.5, build 633a0ea838

Docker-compose version (Only if you use it):  
Not used

Docker Command to start docker-android:  
```
docker run --privileged -d -p, 6080:6080 
  -e "DEVICE=Samsung Galaxy S6
-e APPIUM=true
budtmo/docker-android-x86-9.0:1.8-p12
```
  
## Expected Behavior
Can rotate emulator, and it's working ok.

## Actual Behavior
Observed this behavior:
- Random rectangular black areas appear when rendering page content in Chrome browser inside the emulator
- When the emulator is rotated with the emulator's UI, it dies with this message:
```
ERROR:zygote_host_impl_linux.cc(89)] Running as root without --no-sandbox is not supported
```

## Workaround

Run emulator as a non-root user.

**Comment by doomzhou** [comment]
`export QTWEBENGINE_DISABLE_SANDBOX=1`

from
[link](https://superuser.com/questions/1498614/android-emulator-stops-while-pressing-buttons-on-device-frame)

**Comment by budtmo** [comment]
Hi @doomzhou , 

Do you want to create a PR for your solution? You can add that environment variable directly to the Dockerfile.

**Comment by budtmo** [comment]
it should be fixed in release [1.9-p0](https://github.com/budtmo/docker-android/releases/tag/1.9-p0)
